### ISSUE TITLE
question about reproducibility

### ISSUE THREAD

**Issue by Liyw979** [krumo/Domain-Adaptive-Faster-RCNN-PyTorch#18]
Hi, 这篇论文 https://arxiv.org/abs/1812.04798 里面认为 da-faster-rcnn的ins level会降低map
我几个月前做的时候加入ins level也发现了这个情况
这个你怎么看呢？

**Comment by krumo** [comment]
Hi, if I remember correctly, instance level adaptation is sensitive to the scale value of Gradient Reverse Layer(aka MODEL.DA_HEADS.DA_INS_GRL_WEIGHT in this repo). A large scale value will impair the feature alignment and the final performance. If you find the performance drops with the default scale value, I would recommend you to try a smaller one. In my experience, generally instance-level adaptation could help as long as you choose the correct hyperparameters.

**Comment by Liyw979** [comment]
thanks
